source/includes/_webhooks.md

```diff
@@ -581,17 +581,13 @@ This event is fired two times in the lifecycle of a dividend post request.
         "currency": "GBP",
         "amount": 100
       },
-      "payments": [
-        {
-          "clientId": "clientId",
-          "accountType": "GIA",
-          "payment": {
-            "currency": "GBP",
-            "amount": 100
-          },
-          "status": "DISTRIBUTED"
-        }
-      ],
+      "paymentsSummary": {
+        "totalPayments": 10,
+        "totalFailed": 1,
+        "totalReceived": 2,
+        "totalDistributed" : 3,
+        "totalPending": 4
```

Comment:
it would be good if we could get some kind of description corresponding to each of these fields - a 'batch summary model'? on the _bulk-payments screen that links back here?

+1: also, not too sure a client would be interested in the 'totalReceived' field - that feels more like an internal thing - remove on dto?

Reply:
The payments summary is exposed only on the webhook body, the other endpoints will contain the payments list. I will add the description at webhook level.